deploy/helm/edot-collector/kube-stack/values.yaml

```diff
@@ -935,12 +935,12 @@ instrumentation:
     type: parentbased_traceidratio # Sampler type
     argument: "1.0" # Sampling rate set to 100% (all traces are sampled).
   java:
-    image: docker.elastic.co/observability/elastic-otel-javaagent:1.0.0
+    image: docker.elastic.co/observability/elastic-otel-javaagent:1.3.0
   nodejs:
-    image: docker.elastic.co/observability/elastic-otel-node:0.4.1
+    image: docker.elastic.co/observability/elastic-otel-node:1.0.0
   dotnet:
     image: docker.elastic.co/observability/elastic-otel-dotnet:edge
```

Comment:
Any reasons why we do not use versioned tags here as done for the other SDKs?

Reply:
We'll have a look at the tagging process for the release. Does this block merging this PR? Or can we follow up by including a version once we resolve the process during the build? Do we need to bump this manually after every release? Would `latest` not also be suitable? I'm not familiar with how these images are used.

Reply:
> Does this block merging this PR? Or can we follow up by including a version once we resolve the process during the build?

I'd prefer to update all versions within this PR, if feasible, considering that we need to backport it as well. Are we also sure that `edge` points to the latest SDK anyway?

> Do we need to bump this manually after every release? 

These configuration files are bundled with the Elastic Agent, so they follow the regular Elastic stack release cycle. We should just ensure that the latest active stack versions are using the latest SDKs.

> Would latest not also be suitable?

I would avoid using latest to maintain consistency with the versioned tags used for other SDKs. Additionally, it can make debugging harder if an issue arises.

Reply:
Okay, thanks @girodav. I've identified the fix on the release flow, so hopefully we'll have a correctly tagged image today, and we can update this to use that tag.

> These configuration files are bundled with the Elastic Agent, so they follow the regular Elastic stack release cycle. We should just ensure that the latest active stack versions are using the latest SDKs.

Perhaps we can automate that part at some stage.

Reply:
@trentm There's now a `docker.elastic.co/observability/elastic-otel-dotnet:1.0.1` image if you want to use that tag as part of this PR.

Reply:
```suggestion
    image: docker.elastic.co/observability/elastic-otel-dotnet:1.0.1
```